### PR TITLE
Additional OpenMapSurfer layers

### PR DESCRIPTION
**Pull request by aoles**
Add remaining OpenMapSurfer layers ([preview](https://heigit.org/map-tiles/)).

### CODE REVIEW

**Comment by brunob** [comment]
Thx for this @aoles just a small comment (see inline diff) GTM for me :)

**Comment by aoles** [comment]
Thanks @brunob ! I didn't find the comment you mentioned though, am I missing sth?

File: leaflet-providers.js [hunk 1]
```diff
@@ -56,7 +56,7 @@
 				if (attr.indexOf('{attribution.') === -1) {
 					return attr;
 				}
-				return attr.replace(/\{attribution.(\w*)\}/,
+				return attr.replace(/\{attribution.(\w*)\}/g,
```
Comment (brunob) [inline leaflet-providers.js]: Could you explain why you changed this replace ?

**Comment by brunob** [comment]
@aoles https://github.com/leaflet-extras/leaflet-providers/pull/322/files/156398a2cad8661b5605e186607f5bda38471d78#r283799426 :)

File: leaflet-providers.js (hunk 1 above)
Comment (aoles) [reply leaflet-providers.js]: I had to change it in order to be able to specify more than one attribution reference such as [`'{attribution.OpenMapSurfer}{attribution.OpenStreetMap}'`](https://github.com/leaflet-extras/leaflet-providers/blob/156398a2cad8661b5605e186607f5bda38471d78/leaflet-providers.js#L209). The original implementation would only replace the first one `{attribution.OpenMapSurfer}` leaving `{attribution.OpenStreetMap}` untouched.

Comment (brunob) [reply leaflet-providers.js]: 'k thx for the explaination @jieter any thoughts about this ?

Comment (jieter) [reply leaflet-providers.js]: looks good to me!

File: leaflet-providers.js [hunk 2]
```diff
@@ -200,14 +200,47 @@
...
 				variant: 'roads',
-				attribution: 'Imagery from <a href="http://giscience.uni-hd.de/">GIScience Research Group @ University of Heidelberg</a> &mdash; Map data {attribution.OpenStreetMap}'
+				attribution: 'Imagery from <a href="http://giscience.uni-hd.de/">GIScience Research Group @ University of Heidelberg</a> | Map data '
 			},
 			variants: {
-				Roads: 'roads',
+				Roads: {
+				  options: {
+						variant: 'roads',
+						attribution: '{attribution.OpenMapSurfer}{attribution.OpenStreetMap}'
```
Comment (jieter) [inline leaflet-providers.js]: mixing tabs/spaces here.

Comment (aoles) [reply leaflet-providers.js]: Good catch, fixed in https://github.com/leaflet-extras/leaflet-providers/pull/322/commits/36649b1c4b0222edb9b332cc9566efc295f2ec64.

**Comment by brunob** [comment]
Thx again @aoles i've just released 1.7.0 version for this :)

**Comment by aoles** [comment]
Awesome, thanks a lot for your prompt review and merge!

**Pull request merged**